src/lib/openapi/spec/bulk-registration-schema.ts

```diff
@@ -20,29 +22,48 @@ export const bulkRegistrationSchema = {
                     },
                 },
             },
+            example: ['unleash-edge'],
         },
         appName: {
+            description:
+                'The name of the application that is evaluating toggles',
             type: 'string',
+            example: 'Ingress load balancer',
         },
         environment: {
+            description: 'Which environment is the application running in',
             type: 'string',
+            example: 'development',
         },
         instanceId: {
+            description: 'A unique identifier for the application',
             type: 'string',
+            example: 'Random UUID',
```

Comment:
I suspect that the example here is more of a description than an example of what we would usually expect. If so, would it make sense to do something like this?

```suggestion
            description: 'A unique identifier for the application. Commonly a UUID.',
            type: 'string',
            example: 'AD4F7B4C-0C0C-4AF1-B3C4-95C174E28227',
```